### ISSUE TITLE
"Compiling RuleSet failed: Properties generator are unknown" error

### ISSUE THREAD

**Issue by shameleo** [JetBrains/svg-sprite-loader#428]
**Do you want to request a feature, report a bug or ask a question?**
Bug
**What is the current behavior?**
When I use [webpack 5 Asset Modules feature](https://webpack.js.org/guides/asset-modules/) and `svg-sprite-loader` **at the same time**  I get error mentioned in title.
**If the current behavior is a bug, please provide the steps to reproduce, at least part of webpack config with loader configuration and piece of your code.**
```
module: {
    rules: [
        // ...
        {
            test: /\.(woff|jpg|png|gif)$/,
            type: 'asset/resource',
            generator: {
                filename: `assets/[name]${bldHash}.[ext]`
            }
        },
        {
            test: /\.svg$/,
            loader: 'svg-sprite-loader',
            options: {
                extract: true,
                // ...
            }
        }
    ]
},
```

**Please tell us about your environment:**

- Node.js version: v12.18.3
- webpack version: 5.11.1
- svg-sprite-loader version: 5.2.1

**Other information**
Related issue: https://github.com/vuejs/vue-loader/issues/1729
Fix is really simple: 
**/lib/utils/get-matched-rule-5.js**
```
const ruleSetCompiler = new RuleSetCompiler([
  // ...
  new BasicEffectRulePlugin('generator'),   // add this
  // ...
]);
```
It worked for me. Anyway, I have no idea, what I'm doing (why loader depends on options which are not even related to it?), so I'd like not to create PR and let you to fix it consciousnessly.

**Comment by bernhardberger** [comment]
Is this fixed yet?

**Comment by lauraferrandof** [comment]
Any news on this bug? I'm still getting it with the latest versions of webpack and svg-sprite-loader 😢

**Comment by infabo** [comment]
Bump!

**Comment by tooltonix** [comment]
When will #454 be available in a new version or when will there be a new version? i need these changes urgently.

**Comment by ekasprzyk** [comment]
Running on 6.0.7 and still getting this issue.  Has the fix been released?  (I can't find it inside the 6.0.7 version comparison with 5.*)

**Comment by d3x42** [comment]
@ekasprzyk Please try 6.0.8
